python/cuml/_thirdparty/sklearn/preprocessing/_data.py

```diff
@@ -1072,41 +1072,43 @@ def maxabs_scale(X, *, axis=0, copy=True):
 
 
 class RobustScaler(TransformerMixin, BaseEstimator):
-    """Scale features using statistics that are robust to outliers.
+    """
+    Scale features using statistics that are robust to outliers.
```

Comment:
As below if we can avoid modifying the files under `_thirdparty/sklearn` (which are from sklearn directly), we'll make our lives easier as we update in the future and track which code came from us vs. original... any options to avoid this divergence without a million warnings? A flag or setting?

Reply:
We can maybe create some kind of a hack to make Pydoc parse a separate file with definition-only functions and classes as such :

```
def my_func():
    """pydoc"""
    pass

class my_class:
    """pydoc"""

    def method():
        """pydoc"""
        pass
```

But is it really worth it? As despite best efforts to minimize diffs, there's already a fair amount of them in `_thirdparty/sklearn`. Keeping track of changes should be possible thanks to the [comparison between current gpu-preproc branch and 0.23.1-reduced branch](https://github.com/viclafargue/scikit-learn/compare/0.23.1-reduced...viclafargue:gpu-preproc) containing the sklearn files from version 0.23.1 under the same architecture.

Reply:
I undid the change since it doesn't actually cause a warning. Still worth following up on Victors question

Reply:
Let's leave it out for this release and revisit in 0.17 as we plan to get out of experimental status.